src/test/java/io/cdap/plugin/zendesk/source/batch/ZendeskBatchSourceConfigTest.java

```diff
@@ -322,4 +332,129 @@ public void testGetSchemaInvalidJson() {
         .getCauses().get(0).getAttribute(CauseAttributes.STAGE_CONFIG));
     }
   }
-}
+
+  @Test
+  public void testConfigurePipelineWithInvalidBasicParam() {
+    ZendeskBatchSourceConfig config = new ZendeskBatchSourceConfig(
+      "",
+      "email@test.com",
+      "apiToken",
+      "subdomain",
+      "Groups",
+      "",
+      "2019-01-01T23:01:01Z",
+      "2019-01-01T23:01:01Z",
+      "satisfactionRatingsScore",
+      20,
+      300,
+      300,
+      "https://%s.zendesk.com/api/v2/%s",
+      "");
+    try {
+      ZendeskBatchSource zendeskBatchSource = new ZendeskBatchSource(config);
+      pipelineConfigurer = new MockPipelineConfigurer(null);
+      zendeskBatchSource.configurePipeline(pipelineConfigurer);
+      Assert.fail("Reference name should be valid");
```

Comment:
nit: "Reference name should be valid" -> "Reference name validation should fail".
Similar change for other error msgs in this file